src/store/selectors.ts

```diff
@@ -134,17 +134,32 @@ export const getSpread = createSelector(
     },
 );
 
+export const getSpreadInPercentage = createSelector(
+    getOpenBuyOrders,
+    getOpenSellOrders,
+    (buyOrders, sellOrders) => {
+        if (!buyOrders.length || !sellOrders.length) {
+            return new BigNumber(0);
+        }
+
+        const lowestPriceSell = sellOrders[sellOrders.length - 1].price;
+        const highestPriceBuy = buyOrders[0].price;
+        const spread = lowestPriceSell.minus(highestPriceBuy);
+        return spread.dividedBy(lowestPriceSell).multipliedBy(100);
```

Comment:
PP: This has a lot of duplication with `getSpread`. Maybe we can compute the percentage from the `getSpread` and `getOpenSellOrders` selectors? Or just make one selector that returns a pair?

Reply:
I'd used the `getSpread` and `getOpenSellOrders`. I think making one selector that returns a pair it's not too easy to read as having just two selectors for different things